TypewiseAlert.Test/TypewiseAlertTest.cs

```diff
@@ -3,13 +3,43 @@
 
 namespace TypewiseAlert.Test
 {
-  public class TypewiseAlertTest
-  {
-    [Fact]
-    public void InfersBreachAsPerLimits()
+    public class TypewiseAlertTest
     {
-      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+        [Fact]
+        public void ClassifyTemperatureBreachAsNormal()
+        {
+            Assert.True(CheckBreachLevel.ClassifyTemperatureBreach(BMSConstants.CoolingType.HiActiveCooling, 40) == BMSConstants.BreachType.Normal);
+        }
+        [Fact]
+        public void ClassifyTemperatureBreachAsLow()
+        {
+            Assert.True(CheckBreachLevel.ClassifyTemperatureBreach(BMSConstants.CoolingType.PassiveCooling, -1) == BMSConstants.BreachType.TooLow);
+        }
+
```

Comment:
Test Cases for  Email Notification and Controller Notification ?

Reply:
added the test cases for email and controller notification